dspace/config/spring/api/discovery.xml

```diff
@@ -865,8 +865,11 @@
         <!--The sort filters for the discovery search-->
         <property name="searchSortConfiguration">
             <bean class="org.dspace.discovery.configuration.DiscoverySortConfiguration">
+                <!--The default sort filter to use for the initial workspace loading-->
+                <property name="defaultSortField" ref="sortDateAccessioned" />
```

Comment:
@buithaihai thank you so much for your contribution. It would help a lot many users. 
Can you consider `lastModified` as the default sort field and also proposing `copyField` an addition to search Solr schema?